schemas/config/3.0.schema.json

```diff
@@ -1,2600 +1,2869 @@
+																				"enum": [
+																					"messageGroup",
+																					"queue"
+																				]
+																			},
+																			"FifoThroughputLimit": {
+																				"description": "**High-throughput FIFO queues only.** Specifies whether the FIFO queue throughput quota applies to the entire queue or per message group. The `perMessageGroupId` value is allowed only when the value for `DeduplicationScope` is `messageGroup`.",
+																				"type": "string",
+																				"enum": [
+																					"perQueue",
+																					"perMessageGroupId"
+																				]
+																			},
+																			"KmsMasterKeyId": {
+																				"description": "**Applies only to server-side-encryption.** The ID of an AWS managed customer master key (CMK) for Amazon SQS or a custom CMK. While the alias of the AWS-managed CMK for Amazon SQS is always `alias/aws/sqs`, the alias of a custom CMK can, for example, be `alias/MyAlias`.",
+																				"type": "string"
+																			},
+																			"KmsDataKeyReusePeriodSeconds": {
+																				"description": "**Applies only to server-side-encryption.** The length of time, in seconds, for which Amazon SQS can reuse a data key to encrypt or decrypt messages before calling AWS KMS again. An integer representing seconds, between 60 seconds (1 minute) and 86,400 seconds (24 hours).",
+																				"type": "integer",
+																				"minimum": 60,
+																				"maximum": 86400,
+																				"default": 300
+																			},
+																			"SqsManagedSseEnabled": {
+																				"type": "boolean",
+																				"default": false,
+																				"description": "**Applies only to server-side-encryption.** Enables server-side queue encryption using SQS owned encryption keys. Only one server-side encryption option is supported per queue."
+																			}
+																		}
+																	},
+																	"tags": {
+																		"title": "Tags to associate with the queue. Please see https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-queue-tags.html.",
+																		"type": "object",
+																		"additionalProperties": {
+																			"type": "string",
+																			"minLength": 1
+																		}
+																	}
+																}
+															}
+														}
+													},
+													"else": {
+														"if": {
+															"properties": {
+																"driver": {
+																	"enum": [
+																		"memory"
+																	]
+																}
+															}
+														},
+														"then": true,
+														"else": false
+													}
+												}
+											}
+										}
+									}
+								}
+							}
+						}
+					}
+				}
+			}
+		},
+		"tcp": {
+			"type": "object",
+			"description": "Plugin to handle RAW TCP packets.",
```

Comment:
I've removed all instances of "available since" and similar statements, as documentation should always just reflect latest version, and the entire file should be version controlled if you want to document releases independently.